### ISSUE TITLE
Test Example?

### ISSUE THREAD

**Issue by lonnietc** [sirherrbatka/cl-htm#1]
Hello,

I am getting back into Lisp as well as want to experiment with HTM networks  and came across your cl-htm implementation of the Numenta HTM.

I have been researching on lips interpreters/compilers and it seems that SBCL is supposed to be good as well as fast.

With your cl-htm run on SBCL lisp and also do you have any documentation or examples on how to run the code to test?

Thanks and have a great day

**Comment by sirherrbatka** [comment]
No, no example. There are other implementations with better documentation.
